SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf

```diff
@@ -35,11 +37,11 @@
 	
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/SEC/Debt/AssetBackedSecurities/PoolBackedSecurities/">
 		<rdfs:label xml:lang="en">Pool-backed Securities Ontology</rdfs:label>
-		<dct:abstract>Concepts common to asset backed and mortgage backed securities, including pools and structured finance instruments. 
-		Note that in common industry parlance, the term &apos;asset backed security&apos; is used to refer to securities which specifically do not have a mortgage loan pool as their underlying asset. For this reason, this ontology provides a common parent to both asset backed and mortgage backed securities, which is effectively an asset backed security in the broader sense. This has been labeled as &apos;pool backed security instrument&apos; as there is no common industry name for this common parent class, other than &apos;asset backed security&apos;.</dct:abstract>
+		<dct:abstract>This ontology defines concepts that are common to asset-backed and mortgage-backed securities, including pools, as well as structured finance instruments. 
+		Note that in common industry parlance, the term &apos;asset backed security&apos; is used to refer to securities which specifically do not have a mortgage loan pool as their underlying asset. For this reason, this ontology provides a common parent to both asset-backed and mortgage-backed securities, i.e., a pool-backed security, which is effectively an asset backed security in the broader sense.</dct:abstract>
```

Comment:
this seems to be saying that our semantic model is based not on semantics but a judgement of common industry parlance, which is a confusing message. Have we applied that consistently? 
Seems we could avoid confusion by avoiding the confusing names as primary class/property names and keep them as synonyms, with more neutral names in the ontology. In this case "Pool backed" is good but we should consider renaming "asset backed" to "NonMortgageLoanBackedSecurity": a bit unwieldy I know but it reflects the semantics.
Either way we should document our approach more generally in contribution guidelines.

Reply:
@rivettp We could revise the abstract, but I'm against revising the name at this stage.  I'll remove the comment from the abstract since I don't think it is necessary.  It was 'left over' from the original as you can see.

Reply:
I'm less worried about the ontology name (to which the abstract applies) than the class name which is what my suggestion was intended to be for.

Reply:
@rivettp I think the current name is what is used in the industry, and that there is some overloading of the concept we have called pool-backed security which some also refer to as asset-backed security, but the narrower concept is correctly named.  We can confirm that with John and Jeff in two weeks, or on another call, but I would think that we would have egg on our face if we did not have a definition for asset-backed security.